draft-demarco-oauth-status-attestations.md

```diff
@@ -459,7 +459,8 @@ The Status Assertion MUST contain the parameters defined below.
 | --- | --- | --- |
 | **alg** | A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST NOT be set to `none` or to a symmetric algorithm (MAC) identifier. | {{RFC7515}}, {{RFC7517}} |
 | **typ** | It MUST be set to `status-assertion+jwt` when JWT format is used. It MUST be set to `status-assertion+cwt` when CWT format is used. | {{RFC7515}}, {{RFC7517}} and this specification |
-| **kid** | Unique identifier of the Credential Issuer JWK | {{RFC7515}} |
+| **kid** | Unique identifier of the Credential Issuer JWK. It is required when `x5c` or other parameter such as `did` are not used. | {{RFC7515}} |
```

Comment:
seems like it could be required when `did` is present, since `kid` is often part of absolute DID URLs... If its possible to avoid introducing `did` in this spec, I think thats preferable.